ertviz/controllers/ensemble_selector_controller.py

```diff
@@ -73,6 +76,7 @@ def update_ensemble_selector(selected_ensembles, _, elements):
                 element["data"].update(
```

Comment:
This updates ensemble regardless it's selected or not, is it intentional?

Reply:
yes, because it should remove the color if it is not selected

Reply:
Sure, I deleted the comment as I grasped the code in the end :)